azure-iot-device/azure/iot/device/request_response.py

```diff
@@ -10,10 +10,13 @@
 
 
 class Response:
-    def __init__(self, request_id: str, status: int, body: str) -> None:
+    def __init__(
+        self, request_id: str, status: int, body: str, properties: Optional[dict] = None
```

Comment:
>dict [](http://example.com/codeflow?start=76&length=4)

Should this be `dict` or should it be `Dict[str, str]`?

Reply:
probably the second is better as non string things should not be returned on topic